contract/r/gnoswap/pool/pool_factory_param.gno

```diff
@@ -44,47 +44,48 @@ func newPoolParams(
 	}
 }
 
-func (p *createPoolParams) Token0Path() string {
-	return p.token0Path
-}
-
-func (p *createPoolParams) Token1Path() string {
-	return p.token1Path
-}
-
-func (p *createPoolParams) Fee() uint32 {
-	return p.fee
-}
-
-func (p *createPoolParams) TickSpacing() int32 {
-	return p.tickSpacing
-}
-
-func (p *createPoolParams) SqrtPriceX96() *u256.Uint {
-	return p.sqrtPriceX96
-}
+func (p *createPoolParams) SqrtPriceX96() *u256.Uint { return p.sqrtPriceX96 }
```

Comment:
How do you think naming `createPoolParams` as poolParams. It looks like creat function in factory, not struct.

Reply:
hmm.. still ambiguous. how about `poolCreateConfig` or `poolSpecification`?

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/612/commits/abb45ef9c1e5e3b3b0f3449586ede6958bc64254